Classes/Issues/Comments/Details/IssueCommentDetailCell.swift

```diff
@@ -114,6 +125,15 @@ final class IssueCommentDetailCell: IssueCommentBaseCell, ListBindable {
         delegate?.didTapProfile(cell: self)
     }
 
+    @objc func onTapCell() {
+        // Only enable the tap of the cell in VoiceOver;
+        // otherwise the avatar and label work as advertised.
+        // It seems like we need to add a recognizer to the cell
+        // to enable an action while using VoiceOver.
+        guard UIAccessibilityIsVoiceOverRunning() else { return }
+        delegate?.didTapProfile(cell: self)
```

Comment:
Personal preference for me but when it's such a trivial function like this I think it reads better to use an `if`. I personally feel it reads a bit easier!

Reply:
I don't mind; feel free to change it